README.md

```diff
@@ -75,6 +78,7 @@ You can use [site environment variables](https://docs.netlify.com/configure-buil
 | `sentryProject` | The slug of the project name in Sentry. | - |
 | `sentryAuthToken` | Authentication token for Sentry. We recommend this be set as an environment variable (see below). | - |
 | `sentryRelease` | The release ID (a.k.a version). | [COMMIT_REF](https://docs.netlify.com/configure-builds/environment-variables/#git-metadata) |
+| `sentryRepository` | Override the name of the target Sentry repository. | - |
```

Comment:
@lewisjared Same as above, description to "The name of the target Sentry repository", with the default of "Derived from Netlify repository URL"